tests/lib/release-ops.js

```diff
@@ -252,6 +284,36 @@ describe("ReleaseOps", () => {
             });
         });
 
+        it("should create the next stable release following a prerelease", () => {
```

Comment:
Previously, assuming our last prerelease before a stable major release was a release candidate, the stable release's changelog would've only included changes since the release candidate. At that phase of the release, breaking changes are discouraged. Now that we're including breaking changes in the stable release's changelog, I added this test to verify that the version bump won't mistakenly go from `7.0.0-rc.0` to `8.0.0` when it sees a `Breaking:` commit in the changelog for a release following a release candidate.